src/main/java/reactor/netty/resources/PooledConnectionProvider.java

```diff
@@ -634,7 +635,7 @@ void registerClose(PooledRef<PooledConnection> pooledRef, InstrumentedPool<Poole
 
 		PoolKey(SocketAddress holder, int pipelineKey) {
 			this.holder = holder;
-			this.fqdn = holder instanceof InetSocketAddress ? holder.toString() : "null";
+			this.fqdn = holder instanceof InetSocketAddress || holder instanceof DomainSocketAddress ? holder.toString() : "null";
```

Comment:
Should this condition be inverted and `toString()` invoked for all non-null cases instead?

Reply:
let's call `toString` in all cases